stable/datadog/values.yaml

```diff
@@ -163,3 +163,10 @@ rbac:
   serviceAccountName: default
 
 tolerations: []
+
+kube-state-metrics:
+  rbac:
+    create: true
+
+    ## Ignored if rbac.create is true
+    serviceAccountName: default
```

Comment:
Does this actually work? The sections in the code look like:

```
{{- end }}
      serviceAccountName: {{ if .Values.rbac.create }}{{ template "datadog.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
{{ end }}
```

Shouldn't it use something like `.Values.kube-state-metrics.rbac.serviceAccountName` instead?

Reply:
@mattfarina Thanks for your comments.

>.Values.kube-state-metrics.rbac.serviceAccountName instead?

No. .
If I hadn't misunderstand,　```kube-state-metrics.rbac.serviceAccount```  in values.yaml override parameter for kube-state-metrics charts (https://github.com/kubernetes/charts/blob/master/stable/kube-state-metrics/templates/deployment.yaml#L22).


FLY: https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/subcharts_and_globals.md#overriding-values-from-a-parent-chart

Reply:
Ah, yup. It was that I missed the rbac section in the values.yaml file on my first pass.

Reply:
Thanks.